spec/unit/intercom/contact_spec.rb

```diff
@@ -248,6 +248,18 @@
     client.contacts.delete(contact)
   end
 
+  it 'archives a contact' do
+    contact = Intercom::Contact.new('id' => '1')
+    client.expects(:post).with('/contacts/1/archive', {}).returns(contact)
```

Comment:
Since we aren't checking for an expected return value. Think its safe to drop the `.returns`
```suggestion
    client.expects(:post).with('/contacts/1/archive', {})
```